core-rust/core-api-server/src/core_api/conversions/substates/type_info_module.rs

```diff
@@ -10,8 +10,8 @@ pub fn to_api_system_boot_substate(
     substate: &SystemBoot,
 ) -> Result<models::Substate, MappingError> {
     let value = match substate {
-        SystemBoot::V1 { costing_parameters } => models::BootLoaderModuleFieldSystemBootValue::new(
-            to_api_system_costing_parameters(context, costing_parameters)?,
+        SystemBoot::V1(system_parameters) => models::BootLoaderModuleFieldSystemBootValue::new(
```

Comment:
The system boot and vm boot aren't really `type_info_module` substates, perhaps they should be put in `boot_loader_module.rs`?